config/development.js

```diff
@@ -3,7 +3,9 @@ exports.config = {
   common: {
     database: {
       name: process.env.DB_NAME_DEV
-    }
+    },
+    clientAPI: 'https://quote-garden.herokuapp.com/api/v3/quotes',
```

Comment:
This should be added as an Environment Variable. Putting this type of information directly in the repository can be very problematic and prone to security issues.